misc/config_definitions.json

```diff
@@ -4992,6 +4992,13 @@
             "type": "integer",
             "default": 161
         },
+        "snmp.timeout": {
+            "group": "poller",
+            "section": "snmp",
+            "order": 1,
+            "type": "integer",
+            "default": 1
```

Comment:
I think the default should be null

Reply:
Doc says 1 second :-)

https://docs.librenms.org/Support/Configuration/

Reply:
1 second is the default that net-snmp sets when none is specified.
If it is null or 0 it won't add `-t`  to the command.  Having it set to 1 will add `-t 1` to every command call.  It is a little cleaner to not have that specified.

Reply:
actually, it would be clearer if it was 1 in the ui.   It would be better the code omits timeout 1